src/ScottPlot5/ScottPlot/Plottables/Heatmap.cs

```diff
@@ -22,27 +22,104 @@ public class Heatmap : IPlottable
         public bool IsVisible { get; set; } = true;
         public IAxes Axes { get; set; } = Axis.Axes.Default;
         public IColormap Colormap { get; set; } = new Viridis();
+
+        /// <summary>
+        /// Indicates position of the data point relative to the rectangle used to represent it.
+        /// An alignment of upper right means the rectangle will appear to the lower left of the point itself.
+        /// </summary>
         public CellAlignment CellAlignment { get; set; } = CellAlignment.Center;
+
+        /// <summary>
+        /// If defined, the this rectangle sets the boundaries of colormap data.
+        /// Note that the actual colormap area is 1 cell larger than this rectangle.
+        /// </summary>
         public CoordinateRect? Extent { get; set; }
+
+        /// <summary>
+        /// By default heatmaps are drawn in the same vertical orientation as vectors: row 0 is at the top.
+        /// If this is true, the vertical orientation will be flipped so row 0 is on the bottom.
+        /// </summary>
         public bool FlipVertically { get; set; } = false;
+
+        /// <summary>
+        /// If true, the heatmap will be blurred.
+        /// If false, the heatmap will appear as individual rectangles with sharp edges.
+        /// </summary>
```

Comment:
I wouldn't call this blurring, it's interpolation. We're making up data rather than obscuring it.